sumologic/sumologic_cse_match_list_item.go

```diff
@@ -0,0 +1,158 @@
+package sumologic
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func (s *Client) GetCSEMatchListItem(id string) (*CSEMatchListItemGet, error) {
+	data, _, err := s.Get(fmt.Sprintf("sec/v1/match-list-items/%s", id))
+	if err != nil {
+		return nil, err
+	}
+
+	if data == nil {
+		return nil, nil
+	}
+
+	var response CSEMatchListItemResponse
+	err = json.Unmarshal(data, &response)
+	if err != nil {
+		return nil, err
+	}
+
+	return &response.CSEMatchListItemGet, nil
+}
+
+func (s *Client) GetCSEMatchListItemsInMatchList(MatcListId string) (*CSEMatchListItemsInMatchListGet, error) {
+
+	request := CSEMatchListItemsRequest{
+		ListIds: []string{MatcListId},
+	}
+
+	data, _, err := s.GetWithPayload("sec/v1/match-list-items", request)
```

Comment:
I think we should send GET params as query params rather than through the body like we do elsewhere.

Reply:
my bad, for some reason I read the api spec wrongly. done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/353/commits/dda976e0c3b3134f2ed3736bdaa8a69b129a6609